tasks/docker-compose.yml

```diff
@@ -0,0 +1,28 @@
+---
+- name: Check current docker-compose version.
+  command: "{{ docker_compose_path }} --version"
+  register: docker_compose_vsn
+  check_mode: false
+  changed_when: false
+  failed_when: false
+
+- set_fact:
+    docker_compose_current_version: "{{ docker_compose_vsn.stdout | regex_search('(\\d+(\\.\\d+)+)') }}"
+  when: docker_compose_vsn.stdout is defined
+
+- name: Delete existing docker-compose version if it's different.
```

Comment:
require sudo

Reply:
Added the required sudo permission statement